app/install/kubernetes.md

```diff
@@ -66,37 +74,49 @@ Kong can easily be provisioned to Kubernetes cluster using the following steps:
     $ kubectl create -f postgres.yaml
     ```
 
-4. **Deploy Kong**
+4. **Prepare database**
+
+    Using the `kong_migration_<postgres|cassandra>.yaml` file from this repo,
+    run the migration job, jump to step 5 if Kong backing databse is up–to–date:
```

Comment:
"and jump to step 5 if Kong database is up-to-date"; also, is "up-to-date" the best term here? Maybe "ready" instead?